templates/frontOffice/modern/I18n/fr_FR.php

```diff
@@ -116,7 +116,7 @@
   'Max price' => 'Prix max',
   'Min price' => 'Prix min',
   'My Addresses' => 'Mes adresses',
-  'My Profile' => 'Mon Profil',
+  'My profile' => 'Mon profil',
```

Comment:
If you change the source phrase ("My Profile" in English) you must also change it in the template {intl}